tests/game/test_buildings.py

```diff
@@ -173,21 +174,58 @@ def test_build_tear(s, p):
 
 
 @game_test(timeout=60)
-def test_tree_production(s, p):
-	"""Check whether trees produce wood"""
+def test_tree_wood_production(s, p):
+	"""
+	Check whether trees produce wood
+	"""
 	settlement, island = settle(s)
+
+	# create building tree at coordinates (30,35)
```

Comment:
That comment doesn't add anything. I don't claim to be an export when writing comments, but typically comments should explain the why, and not the how.

This one says the same as the code below (when I strip some syntax). `Build tree at 30, 35`